huxley/www/js/components/AdvisorAssignmentsView.js

```diff
@@ -230,51 +236,17 @@ var AdvisorAssignmentsView = React.createClass({
     var confirm = window.confirm("By pressing okay you are committing to the financial responsibility of each assignment. Are you sure you want to finalize assignments?");
     var school = CurrentUserStore.getCurrentUser().school;
     if (confirm) {
-      this.setState({loading: true});
-      ServerAPI.updateSchool(school.id, {assignments_finalized: true})
-        .then(this._handleFinalizedSuccess, this._handleError);
+      var delta = {assignments_finalized: true};
```

Comment:
You can just inline this into the function call, no need for the separate binding:
```js
CurrentUserActions.updateSchool(school.id, {
  assignments_finalized: true,
});
```